<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>311</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>Bienvenida</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrese su nombre:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="name">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="age">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>60</y>
       <width>113</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Ingrese su edad:</string>
     </property>
    </widget>
    <widget class="QComboBox" name="comboBox_ocupacion">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>181</width>
       <height>23</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Seleccione su ocupacion:</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Abogado</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Fisico</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Estudiante</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Ingeniero</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Otro</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>311</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Genero</string>
    </property>
    <widget class="QRadioButton" name="radioButton_masculino">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Masculino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_femenino">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Femenino</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_otro">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>30</y>
       <width>100</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Otro</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButton_bienvenida">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Saludo</string>
    </property>
   </widget>
   <widget class="QListWidget" name="list_bienvenida">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>311</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
